identity/backend/src/main/java/io/camunda/identity/config/IdentityPresets.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(value = "camunda.presets.identity")
```

Comment:
let's rather call it `camunda.identity` without presets. We have used `presets` in the past to define parts of the environment, so like a configuration for a whole app like Operate

Reply:
If there is a problem with matching the property path without going into our global configuration classes, an alternative could be `camunda.identity.init` or initialization